src/Movie/Movie.js

```diff
@@ -2,14 +2,33 @@ import React from 'react'
 import './Movie.scss'
 
 const Movie = ({ movie }) => {
+  const genres = movie.genres.map(genre => {
+    return (
+      <li key={genre}>{ genre }</li>
+    )
+  })
+
+  const convertNumber = (num, type) => {
+    return (
+      num > 0 && <p><b>{ type }:</b> ${new Intl.NumberFormat('en-US').format(num)}</p>
+    )
+  }
+
   return (
     <section className='movie-section' style={{ background: `url(${ movie.backdrop_path })`}}>
       <article className="movie-info">
         <img src={ movie.poster_path } alt="movie-poster" className="movie-poster"></img>
         <div className="movie-text">
-          <h2>{movie.title}</h2>
-          <p><b>Average Rating:</b> {movie.average_rating.toFixed(1)}</p>
-          <p><b>Release Date:</b> {new Date(movie.release_date).toDateString()}</p>
+          <h2>{ movie.title }</h2>
+          <h3>{ movie.tagline }</h3>
+          <p>{ movie.overview }</p>
+          <p><b>Average Rating:</b> { movie.average_rating.toFixed(1) }</p>
+          <p><b>Release Date:</b> { new Date(movie.release_date).toDateString() }</p>
+          <p><b>Runtime:</b> { movie.runtime } minutes</p>
+          { convertNumber(movie.budget, 'Budget') }
```

Comment:
I'm so glad we took the time to experiment with this.  I think it makes this dryer and more dynamic.